routes/users.js

```diff
@@ -1,32 +1,12 @@
 const express = require('express'),
       router = express.Router(),
       settings = require('../app_settings'),
-      UserModel = require('../models/UserModel');
+      UserModel = require('../models/UserModel'),
+      authChecker = require('../helpers/authChecker');
 
-router.get('/', function (req, res) {
-    UserModel.find({}, function (err, users) {
-        if (err) {
-            console.log(`User request error: ${err}`);
-            res.redirect(`/users`);
-        } else {
-            res.render('users/index', {
-                users: users,
-                pageTitle: "Users",
-            });
-        }
-    });
-});
-
-router.get('/new', function (req, res) {
-    res.render('users/new', {
-        pageTitle: 'Registration',
-    });
-});
-
-router.get('/:userId', function (req, res) {
-    let user = {};
+router.get('/', authChecker, function (req, res) {
```

Comment:
`/users` should return a user index. `/users/:userID` is the RESTful way to handle this. Need to adjust.